crypto/evp/m_sigver.c

```diff
@@ -75,6 +75,14 @@ static int do_sigver_init(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
         return 1;
     if (!EVP_DigestInit_ex(ctx, type, e))
         return 0;
+    if (ctx->pctx->pmeth->flags & EVP_PKEY_FLAG_DIGEST_CUSTOM) {
```

Comment:
`EVP_PKEY_FLAG_DIGEST_CUSTOM` is unnecessary, it should be enough to check that `digest_custom` is non-NULL

Reply:
Ahh, makes sense